src/com/algosolution/AlgoSolution.java

```diff
@@ -0,0 +1,23 @@
+package com.algosolution;
+
+
+
+public class AlgoSolution {
+    public static void FIND(int Start, int End, String Text) {
```

Comment:
You should learn about naming convention while coding, you don't use capital letter to start variable name. Kindly change Start, End and Text